package.json

```diff
@@ -30,16 +30,19 @@
     "lint": "tslint -t stylish --project \"tsconfig.json\"",
     "test": "npm run test-only",
     "test-only": "jest --coverage",
-    "watch": "tsc -w -p tsconfig.build.json"
+    "watch": "tsc -w -p tsconfig.build.json",
+    "webpack": "webpack"
```

Comment:
I added this because I had been doing some development in the graphics lab, which has npm but not yarn. Yarn automatically translates rules like this, but npm doesn't.